### ISSUE TITLE
mpv configure failure with waf 2.0.1

### ISSUE THREAD

**Issue by jpalus** [waf-project/waf#2054]
[check_waf_version() in waflib/Configure.py](https://github.com/waf-project/waf/blob/210874d297147db24637e8b7751c2f0f9b98657d/waflib/Configure.py#L375) does not seem to be updated for version 2.0.1:

```python
def check_waf_version(self, mini='1.8.99', maxi='2.0.0', **kw):
```

resulting in mpv build failure which does not specify maxi explicitly:

    Checking for waf version in 1.8.4-2.0.0  : waf version should be at most 33554432 (33554688 found)

**Comment by ita1024** [comment]
Use the version that comes with mpv.

**Comment by jpalus** [comment]
Why is the issue marked as invalid if you did fix it in 6d18a0e? mpv was just an example of broken behavior in new waf, it applies to other apps as well. But anyway I won't use waf provided by mpv to build package provided by distribution.
